### PR TITLE
Import KETTLE, a pipeline to get GCS test results into BigQuery.

### PR DESCRIPTION
**Pull request by rmmh**
This was running on my workstation for a while, but I've cleaned it up and added tests now!

### CODE REVIEW

**Comment by spxtr** [comment]
Please write a `BUILD` file for this, it will probably look pretty similar to the [test_history BUILD file](https://github.com/kubernetes/test-infra/blob/master/jenkins/test_history/BUILD). That way we don't need a whole new job for it.

**Comment by rmmh** [comment]
Added a BUILD file.

File: prow/config.yaml [hunk 1]
```diff
@@ -189,6 +189,7 @@ presubmits:
       containers:
       - image: gcr.io/k8s-testimages/gubernator:0.2
 
+
```
Comment (spxtr) [inline prow/config.yaml]: no need for the newline I guess

**Comment by spxtr** [comment]
This is pretty hard to review because has quite a bit pulled from test_history and gubernator but then modified in subtle ways. Would it be possible to share some of the code?

**Comment by rmmh** [comment]
Only GCSClient and parse_junit are really shared here. I'd like to eliminate the majority of test_history in the future.

**Comment by spxtr** [comment]
/lgtm

What's the plan to make it run regularly? A periodic job on prow would work, as would some other ideas.

**Comment by rmmh** [comment]
This currently depends on a large local database, so I'm going to put it in a VM.

**Comment by spxtr** [comment]
You could make a PV and include it in the job spec.

**Comment by spxtr** [comment]
Probably not super necessary though. A VM is probably easier to debug anyway.

**Pull request merged**